dcos/dcos-ui
# fix: parse scaling.instances 0

This is a followup on #3809 turned out the fix didn't address issue in parser, so `0` wouldn't be read out.

## Testing

1. Create a pod with 0 instances.
2. Click editing the pod
3. Verify it still has 0 instances
